pkg/app/piped/planpreview/builder.go

```diff
@@ -135,8 +136,22 @@ func (b *builder) build(ctx context.Context, id string, cmd model.Command_BuildP
 		return nil, nil
 	}
 
+	// Prepare source code at the head commit.
+	// This clones the base branch and merges head banch into it for correct data.
+	// Because new changes were added into base branch after head branch was checked out.
```

Comment:
```suggestion
	// Prepare source code at the head commit.
	// This clones the base branch and merges the head branch into it for correct data.
	// Because new changes might be added into the base branch after the head branch had checked out.
```
typo and nits.

Reply:
👍